pkg/controller/rds/dbinstance/setup.go

```diff
@@ -424,6 +446,53 @@ func compareTimeRanges(format string, expectedWindow *string, actualWindow *stri
 	return false, nil
 }
 
+func compareVPCSecurityGroupIDs(cr *svcapitypes.DBInstance, out *svcsdk.DBInstance) bool {
```

Comment:
The return value is easy to misunderstand. Maybe change it to `isEqualVPCSecurityGroup`.

Reply:
Nice to hear. I was initially unsure if or how I should use this more commonly used return logic in this existing resource.
Hopefully the newly committed change compromises not to much.